source/_integrations/azure_event_hub.markdown

```diff
@@ -24,108 +25,27 @@ Once you have the name of your namespace, instance, Shared Access Policy and the
 
-## Configuration
+{% include integrations/config_flow.md %}
 
-Add the following lines to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-azure_event_hub:
-  event_hub_namespace: NAMESPACE_NAME
-  event_hub_instance_name: EVENT_HUB_INSTANCE_NAME
-  event_hub_sas_policy: SAS_POLICY_NAME
-  event_hub_sas_key: SAS_KEY
-  filter:
-    include_domains:
-    - homeassistant
-    - light
-    - media_player
-```
-
-{% configuration %}
-event_hub_namespace:
-  description: The name of your Event Hub namespace.
-  required: exclusive
-  type: string
-event_hub_instance_name:
-  description: The name of your Event Hub instance.
-  required: true
-  type: string
-event_hub_sas_policy:
-  description: The name of your Shared Access Policy.
-  required: exclusive
-  type: string
-event_hub_sas_key:
-  description: The key for the Shared Access Policy.
-  required: exclusive
-  type: string
-event_hub_connection_string:
-  description: The connection string to your event hub.
-  required: exclusive
-  type: string
-send_interval:
-  description: The interval in seconds should events be sent to the Event Hub.
-  required: false
-  type: integer
-  default: 5
-max_delay:
-  description: The time in seconds after which a message is to be discarded.
-  required: false
-  type: integer
-  default: 30
-filter:
-  description: Filter domains and entities for Event Hub. ([Configure Filter](#configure-filter))
-  required: true
-  type: map
-  default: Includes all entities from all domains
-  keys:
-    include_domains:
-      description: List of domains to include (e.g., `light`).
-      required: false
-      type: list
-    exclude_domains:
-      description: List of domains to exclude (e.g., `light`).
-      required: false
-      type: list
-    include_entity_globs:
-      description: Include all entities matching a listed pattern (e.g., `sensor.weather_*`).
-      required: false
-      type: list
-    exclude_entity_globs:
-      description: Exclude all entities matching a listed pattern (e.g., `sensor.weather_*`).
-      required: false
-      type: list
-    include_entities:
-      description: List of entities to include (e.g., `light.attic`).
-      required: false
-      type: list
-    exclude_entities:
-      description: List of entities to include (e.g., `light.attic`).
-      required: false
-      type: list
-{% endconfiguration %}
+You can setup [filters](#filter-configuration) through the `configuration.yaml`, any other keys from the old configuration are forwarded to the config flow and stored by Home Assistant and can be deleted from your configuration.
```

Comment:
We normally don't mention old configuration formats. Just document what is available now.